OtripleS.Portal.Web/Models/Teachers/TeacherGender.cs

```diff
@@ -0,0 +1,14 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+// ---------------------------------------------------------------
+
+namespace OtripleS.Portal.Web.Models.Teachers
+{
+    public enum TeacherGender
```

Comment:
Isn't this the same as `StudentGender`? Why not creating just a `Gender` enum and using the same on both places?

Reply:
would you prefer if i changed the gender in the student model ?
that will require changes to the StudentViewService.cs

Reply:
Leave it for now. If only IApiBroker.Teachers.cs, ApiBroker.Teachers.cs and empty Teacher's model are kept, this can be handled within a different PR.